README.rst

```diff
@@ -19,5 +19,5 @@ You can access the AMO stage environment at https://addons.allizom.org/
 You can join our mailing list at: https://mail.mozilla.org/listinfo/dev-addons
 
 .. _`addons.mozilla.org`: https://addons.mozilla.org
-.. _`install docs`: https://addons-server.readthedocs.org/en/latest/topics/install-olympia/docker.html
+.. _`install docs`: https://addons-server.readthedocs.org/en/latest/topics/install/docker.html
```

Comment:
This is in the README though so I don't think that will work? I don't think GitHub would know to link to https://addons-server.readthedocs.org so I don't know where the link would go.